CamMerrill/Shackelford
# Fixed Lab8 cave environment

## Summary
Fixed the invisible cave walls near the digsite for Lab8

## Testing
1. Load Lab8 map
2. Ensure that the cave walls look correct and there aren't any missing
    - the wall around the wood frame
    - the wall at the back corner behind the excavation site
